src/lib/viewers/image/ImageBaseViewer.js

```diff
@@ -167,6 +168,42 @@ class ImageBaseViewer extends BaseViewer {
         this.bindControlListeners();
     }
 
+    /**
+     * Sets the original image width and height on the img element. Can be removed when
+     * naturalHeight and naturalWidth attributes work correctly in IE 11.
+     *
+     * @private
+     * @param {Image} imageEl - The image to set the original size attributes on
+     * @return {Promise} A promise that is resolved if the original image dimensions were set.
+     */
+    setOriginalImageSize(imageEl) {
+        const image = imageEl;
+        const promise = new Promise((resolve, reject) => {
+            // Do not bother loading a new image when the natural size attributes exist
+            if (imageEl.naturalWidth > 0 && imageEl.naturalHeight > 0) {
+                image.originalWidth = imageEl.naturalWidth;
+                image.originalHeight = imageEl.naturalHeight;
+                resolve();
+            } else {
+                const originalImg = new Image();
+                image.originalWidth = 1;
+                image.originalHeight = 1;
```

Comment:
Instead of adding properties directly to an HTML element, it's better to add a data attribute, see: https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes

```
// Set
imageEl.dataset.originalWidth = 1;
imageEl.dataset.originalHeight = 1;

// Get
const originalWidth = imageEl.dataset.originalWidth;
const originalHeight = imageEl.dataset.originalHeight;
```